### ISSUE TITLE
-Wdocumentation false positive on multiple params separated by comma

### ISSUE THREAD

**Issue by oracle-9** [llvm/llvm-project#121798]
The following code, straight from [Doxygen's manual](https://www.doxygen.nl/manual/commands.html#cmdparam), triggers a warning when compiled with `-Wdocumentation`:
```cpp
/** Sets the position.
 *  @param x,y,z Coordinates of the position in 3D space.
 */
void setPosition(double x,double y,double z,double t)
{
}
```
`warning: parameter 'x,y,z' not found in the function declaration [-Wdocumentation]`.

[Godbolt link](https://godbolt.org/z/ccehn9bKs)
